File: php/Car.php
<?php	
    class Car
    {
        private $id;
        private $license;
        private $driver;
        private $passenger;

        public function __construct($license, $driver)
        {
            $this->license = $license;
            $this->driver = $driver;
        }

        public function printDataCar()
        {
            echo "Mi nombre es: {$this->driver->name} y mi licencia es: {$this->license}";
        }
    }
    
?>

File: python/car.py
from account import Account

class Car:
    id = int
    license = str
    driver = Account("", "")
    passenger = int

    def __init__(self, license, driver):
        self.license = license
        self.driver = driver

    def printDataCar(self):
        print(f"License: {self.license} Driver: {self.driver.name}")